fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java

```diff
@@ -185,6 +186,16 @@ public void setPageTitle(int titleId) {
     mPageTitle.postValue(titleId);
   }
 
+  public MutableLiveData<Boolean> isHome() {
+    return isHome;
+  }
+
+  public void setIsHome(boolean isHome) {
```

Comment:
You could add a comment explaining what this does

Reply:
Sure 👍🏼